src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/ProjectBuilder.cs

```diff
@@ -84,7 +84,7 @@ public bool Build (XamarinProject project, bool doNotCleanupOnUpdate = false, st
 
 		public bool Install (XamarinProject project, bool doNotCleanupOnUpdate = false, bool saveProject = true)
 		{
-			return RunTarget (project, "Install", doNotCleanupOnUpdate, saveProject: saveProject);
+			return RunTarget (project, "Build,Install", doNotCleanupOnUpdate, saveProject: saveProject);
```

Comment:
I think we should put this comment here, because I think this is weird enough it needs an explanation.
```suggestion
			//NOTE: since $(BuildingInsideVisualStudio) is set, Build will not happen by default
			return RunTarget (project, "Build,Install", doNotCleanupOnUpdate, saveProject: saveProject);
```